new-dti-website/components/team/TeamAbout.tsx

```diff
@@ -0,0 +1,264 @@
+import { Dispatch, SetStateAction, useState } from 'react';
+import Image from 'next/image';
+import { ibm_plex_mono } from '../../src/app/layout';
+import FA23Members from '../../../backend/src/members-archive/fa23.json';
+import useScreenSize from '../../src/hooks/useScreenSize';
+import RedBlob from '../blob';
+import { LAPTOP_BREAKPOINT, TABLET_BREAKPOINT } from '../../src/consts';
+
+type roleStatistics = {
+  [key: string]: {
+    name: string;
+    color: string;
+    people: number;
+    majors: Set<string>;
+    colleges: Set<string>;
+  };
+};
+
+type PieChartProps = {
+  width: number;
+  height: number;
+  chartSection: string | undefined;
+  setChartSection: Dispatch<SetStateAction<string | undefined>>;
+  roleStats: roleStatistics;
+  allMembers: IdolMember[];
+};
+
+const PieChart: React.FC<PieChartProps> = ({
+  width,
+  height,
+  chartSection,
+  setChartSection,
+  roleStats,
+  allMembers
+}) => {
+  const radius = 175;
```

Comment:
It would be nice to extract these numbers out as constants outside of the component, maybe at the top of the file so they're easier to find?